MapzenSDK/Marker.swift

```diff
@@ -256,7 +256,7 @@ public class PointMarker : GeometricMarker, GenericPointIconMarker {
   /**
    Initializes a marker with a given size.
 
-   - parameter size: The size the marker should be.
+   - parameter s: The size the marker should be.
```

Comment:
Indeed there is - it's just the `s` is now public facing as of 1.0 so it's harder to just change that. The plan is to deprecate these methods in 1.1.0 and then remove in 2.0.0 to clean that up.